renovate.json

```diff
@@ -3,6 +3,7 @@
     "github>energywebfoundation/shared-configs"
   ],
   "lockFileMaintenance": { "enabled": true },
+  "ignorePaths": [".github/workflows"],
```

Comment:
Could you please explain why these are ignored?

Reply:
because currently implemented actions break after upgrading to clowdhaus/argo-cd-action/@v1.14.0 as a result there is always one active PR that is not possible to be merged. And anyway, these things do not relate to production code, so I decided to disable Renovate completely for actions.

Reply:
so you think there is a bug in clowdhaus/argo-cd-action/@v1.14.0?

Reply:
I don't know. Building images fails after this update. After I talked to the devops team, they promised to exclude this in shared configs, but I am still getting updates, so I have decided to reduce this noise and upgrade GHA dependencies manually if needed.